sample/src/main/java/com/twilio/verify/sample/AppApplication.kt

```diff
@@ -50,7 +51,7 @@ class AppApplication : Application() {
     super.onCreate()
     FirebaseApp.initializeApp(this)
     startKoin {
-      androidLogger()
+      androidLogger(level = Level.ERROR)
```

Comment:
Why are we changing this?

Reply:
it was causing an error in my building on main branch, but it didn't in dev. Removed